src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java

```diff
@@ -44,17 +44,7 @@ public FeedbackRankOptionsQuestionDetails() {
 
     @Override
     public List<String> getInstructions() {
-        List<String> instructions = new ArrayList<>();
-
-        if (minOptionsToBeRanked != NO_VALUE) {
-            instructions.add("You need to rank at least " + minOptionsToBeRanked + " options.");
-        }
-
-        if (maxOptionsToBeRanked != NO_VALUE) {
-            instructions.add("Rank no more than " + maxOptionsToBeRanked + " options.");
-        }
-
-        return instructions;
+        return null;
```

Comment:
for now i will allow this to happen. but pls remove this method completely once you are done with all question types

Reply:
As far as I know we show constraint warnings only for rank and distribute points questions (please correct me if I am wrong). Just to clarify should we then remove this method from the base class `FeedbackQuestionDetails` as well ?